build.gradle

```diff
@@ -23,13 +23,12 @@ group = 'redis.clients'
 
 versions.with {
     guava = '18.0'
-    jedis = '2.6.0'
+    jedis = '2.6.2'
```

Comment:
good point about 2.6.3, not sure why i haven't bumped it to 2.6.3. Not sure about 2.7 though...

Ah I know!
v2.6.3
@marcosnils marcosnils *released this 2 days ago* · 292 commits to master since this release